frontend/public/components/overview/index.tsx

```diff
@@ -517,8 +517,13 @@ class OverviewMainContent_ extends React.Component<OverviewMainContentProps, Ove
     });
   }
 
-  getOverviewLabels(items: OverviewItem[]): string[] {
-    return _.flatMap(items, item => _.keys(_.get(item, 'obj.metadata.labels'))).sort();
+  getLabels(items: OverviewItem[]): string[] {
+    const labelSet = new Set<string>();
```

Comment:
Could this just be: 
```suggestion
    const labelSet = new Set<string>( _.flatMap(items, item => _.keys(_.get(item, 'obj.metadata.labels'))));
```

Reply:
It's not as efficient. It's more compact, but I'm not sure it's easier to understand.